bears/vcs/git/GitCommitBear.py

```diff
@@ -121,7 +121,8 @@ def check_shortlog(self, shortlog,
             if not match:
                 yield Result(
                     self,
-                    "Shortlog of HEAD commit does not match given regex.")
+                    "Shortlog of HEAD commit does not match given regex.\n" +
+                    shortlog_regex)
```

Comment:
brackets could confuse the reader to think that these belong to the regex (though they effectively do nothing, it would just be confusing)